schoolbot/cogs/school/meal.py

```diff
@@ -1,16 +1,24 @@
 import asyncio
 
 import neispy
+import os
 from neispy import DataNotFound
 import discord
 from discord.ext import commands
+import aiohttp
 
 
 class Meal(commands.Cog):
     def __init__(self, bot, apikey):
         self.bot = bot
         self.neis = neispy.AsyncClient(apikey)
 
+    async def render_meal_image(self, meals):
+        async with aiohttp.ClientSession() as session:
+            meals = '&meal='.join(meals.split('<br/>'))
```

Comment:
You not resolve this conversation
i think you push in another branch
please check